public/js/lockToggle.js

```diff
@@ -11,7 +11,7 @@ const urlID = urlParams.get('id');
                                 e.target.classList.toggle("bi-unlock");
                                 e.target.classList.toggle("bi-lock");
                         } else {
-                                document.querySelector(`#${fieldName}Error`).textContent = data.message;
+                            document.querySelector(`#${fieldName}Error`).textContent = data.message;
```

Comment:
you could put the selector in a variable instead of selecting from the DOM twice